tools/docker/builder/Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN apt-get update && apt-get install -y \
     binutils \
     clang-format \
     cmake \
+    cppcheck \
```

Comment:
We actually don't use it in this image anymore, so I guess this commit is not needed.

Do we have a Dockerfile for the Alpine image?

Reply:
Indeed it's a mistake. The alpine image is already there and already contains cppcheck (`tools/docker/builder/alpine/Dockerfile`).